SRS without gamma

dtebbs opened the pull request:
Depends on details of clearmatics/libsnark#3 so worth reviewing that first.

AntoineRondelet commented on src/test/simple_test.tcc:
```diff
@@ -0,0 +1,56 @@
+
+    pb.add_r1cs_constraint(r1cs_constraint<FieldT>(g1, x, g2), "g2");
+
+    //                    g_out
```
`g_out` -> `y`?

dtebbs replied:
Thanks.  Should be fixed in latest.

AntoineRondelet commented on zeth-contracts/contracts/Groth16Verifier.sol:
```diff
@@ -36,27 +35,25 @@ contract Groth16Verifier {
     VerifyingKey verifyKey;
 
     event LogVerifier(string);
+    event LogVerifierUint(uint);
```
Where do you use this event? I guess it was for a debugging purpose. Shall we keep it or do you think we ca remove it now if it is not used?

dtebbs replied:
Thanks.  Removed in latest version.

AntoineRondelet commented on src/test/simple_test.cpp:
```diff
@@ -0,0 +1,70 @@
+    libzeth::test::simple_circuit<ppT>(pb);
+
+    // Constraint system
+
```
Remove extra line

dtebbs replied:
should be removed in latest version

AntoineRondelet commented on src/test/simple_test.cpp:
```diff
@@ -0,0 +1,70 @@
+TEST(SimpleTests, SimpleCircuitProof)
+{
+    // Simple circuit
+
```
Remove extra line

dtebbs replied:
removed in latest version

AntoineRondelet commented on src/test/simple_test.hpp:
```diff
@@ -0,0 +1,25 @@
+namespace test
+{
+
+/// Generate a simple test circuit with 1 public input 'y' and auxiliary
```
Let's stick to:
```
/*
 *
**/
```

For comments on multiple lines, rather than `///` which are not used anywhere else in the code base

AntoineRondelet commented on src/test/simple_test.tcc:
```diff
@@ -0,0 +1,58 @@
+    //  /  \
+    //  \  /
+    //   x
+
```
Remove extra line

AntoineRondelet commented on src/test/simple_test.tcc:
```diff
@@ -0,0 +1,58 @@
+    //   g2
+    //  /  \
+    // g1   x
+
```
Remove extra line

AntoineRondelet commented on src/test/simple_test.tcc:
```diff
@@ -0,0 +1,58 @@
+    //                   /   \
+    //                  /     \
+    // g2 + 4.g1 + 2x + 5      1
+
```
Remove extra line

AntoineRondelet commented on src/test/simple_test.hpp:
```diff
@@ -0,0 +1,25 @@
+///   g1 = x * x
+///   g2 = g1 * x
+template <typename ppT>
+void simple_circuit(libsnark::protoboard<libff::Fr<ppT>> &pb);
```
`libsnark::protoboard<libff::Fr<ppT>>` -> `libsnark::protoboard<FieldT>`

AntoineRondelet commented on src/test/simple_test.tcc:
```diff
@@ -0,0 +1,58 @@
+void simple_circuit(libsnark::protoboard<libff::Fr<ppT>> &pb)
+{
+    using namespace libsnark;
+    using FieldT = libff::Fr<ppT>;
```
Let's use `FieldT` as a template, instead of `ppT` (you define it in the `simple_test.cpp` file and use it with the protoboard)

dtebbs replied:
Changed the template parameter

AntoineRondelet commented on src/test/simple_test.cpp:
```diff
@@ -0,0 +1,70 @@
+        r1cs_gg_ppzksnark_generator<ppT>(constraint_system);
+
+    // Proof and verification
+
```
Remove extra line

AntoineRondelet commented on src/test/simple_test.cpp:
```diff
@@ -0,0 +1,70 @@
+
+} // namespace
+
+
```
Remove extra line

AntoineRondelet commented on src/test/simple_test.hpp:
```diff
@@ -0,0 +1,25 @@
+} // namespace test
+} // namespace libzeth
+
+//
```
What is this `//` for?

AntoineRondelet commented on src/test/simple_test.cpp:
```diff
@@ -0,0 +1,70 @@
+using namespace libsnark;
+
+namespace
+{
```
namespace {`

AntoineRondelet replied:
The bracket on the next line follows the newly adopted style conventions.

AntoineRondelet commented on src/test/simple_test.cpp:
```diff
@@ -0,0 +1,70 @@
+namespace
+{
+
+TEST(SimpleTests, SimpleCircuitProof)
```
I'd stick to `TEST(SimpleTests, SimpleCircuitProof) {` and move the bracket up for consistency with other files.

AntoineRondelet replied:
The bracket on the next line follows the newly adopted style conventions.

AntoineRondelet commented on src/test/simple_test.hpp:
```diff
@@ -0,0 +1,25 @@
+namespace libzeth
+{
+namespace test
+{
```
```
namespace libzeth {

namespace test{
```

AntoineRondelet replied:
The bracket on the next line follows the newly adopted style conventions.

AntoineRondelet commented on src/test/simple_test.tcc:
```diff
@@ -0,0 +1,58 @@
+namespace libzeth
+{
+namespace test
+{
```
```
namespace libzeth {

namespace test {
```

AntoineRondelet replied:
The bracket on the next line follows the newly adopted style conventions.

dtebbs commented:
@AntoineRondelet  Thanks for the comments.  I think I've addressed all the points about content.  I've also rebased onto origin/develop and used the reformatting tool, which should address the points about code consistency.
